pkg/controller/workspace/tfc_org.go

```diff
@@ -20,9 +20,21 @@ var (
 	AutoApply = true
 )
 
+type ClientProvider interface {
```

Comment:
This is a bit convoluted I feel but it was the only way I could think of to be able to mock the tfclient. I'd be happy to be shown a better way.